spec/album_spec.rb

```diff
@@ -31,8 +31,14 @@
   end
 
   it 'should be able to be archived' do
-    @album = MusicAlbum.new('12/02/2005', false)
+    @album = MusicAlbum.new('12/02/2005', false, false)
     @album.move_to_archive
     expect(@album.archived).to eq(true)
   end
+
+  it 'should be able to be archived' do
+    @album = MusicAlbum.new('12/02/2005', false, true)
+    @album.move_to_archive
+    expect(@album.archived).to eq(false)
```

Comment:
Previous changes will break the test, therefore this change is necessary.
```suggestion
    @album = MusicAlbum.new('12/02/2005', false, true)
    @album.move_to_archive
    expect(@album.archived).to eq(true)
```